bloom-instance/tfvars.template

```diff
@@ -3,4 +3,32 @@ application_name = "Bloom Housing Instance"
 resource_prefix = "doorway_bloom"
 team_name = "doorway"
 aws_region = "us-west-1"
-sdlc_stage = "dev"
\ No newline at end of file
+sdlc_stage = "dev"
+subnet_map = {
+    "public" = [
+      {
+        az   = "${var.aws_region}a", # us-west-1a
+        cidr = "10.0.0.0/24"
+      },
+      {
+        az   = "${var.aws_region}b", # us-west-1b
```

Comment:
Most of the time it's safe to assume that there is a "b" AZ in a given region.  For whatever reason, however, us-west-2 only has AZs "a" and "c", so we'll want to change the default from "b" to "c".

Reply:
Since I'm explicitly specifying us-west-1 as the region I will leave the AZ as is.

Reply:
Sorry I misunderstood and thought that was for us-west-2 instead of us-west-1. Changed it now.